src/test/java/tests/PriceFiltersWinetimeTest.java

```diff
@@ -0,0 +1,47 @@
+package tests;
+
+import common.TestInit;
+import io.qameta.allure.Description;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.WinetimeHomePage;
+import pages.WinetimeProductPage;
+
+import java.util.List;
+
+import static com.codeborne.selenide.Selenide.switchTo;
+import static utils.ElementUtil.getListOfStrings;
+
+public class PriceFiltersWinetimeTest extends TestInit {
+
+    HomePage homePage = new HomePage();
+    WinetimeHomePage winetimeHomePage = new WinetimeHomePage();
+    WinetimeProductPage winetimeProductPage = new WinetimeProductPage();
+
+    @Test(description = "Check price filters on WINETIME shop page")
+    @Description("SF2-29")
+    public void checkPriceFilters() {
+        String minPrice = "1000";
+        String maxPrice = "2000";
```

Comment:
move on level higher and make constants
e.g. `private static final String MIN_PRICE = "1000"`